src/app/offre-de-stage-detaillee/offre-de-stage-detaillee.component.html

```diff
@@ -0,0 +1,107 @@
+<div class="container">
+    <div class="title">
+      <p>Offre de stage</p>
+      <h1>Integrateur web junior</h1>
+      <h2 class="square">Acolyte communication</h2>
+    </div>
+  
+    <div class="icons">
+      <a href=""><mat-icon class="check-icon">check</mat-icon></a>
+      <a href=""><mat-icon class="edit-icon">edit</mat-icon></a>
+      <a href=""><mat-icon class="close-icon">close</mat-icon></a>
+    </div>
```

Comment:
Comme on travaille avec le `RouterModule` d'Angular, il est important d'utilisant ce module pour naviguer à travers l'application et non de renvoyer l'utilisateur vers une nouvelle page web. Et pour ce qui est du `close-icon`, est-ce qu'il n'est pas supposé supprimer l'offre de stage? Si c'est le cas, c'est mieux d'utiliser un `button`.

 ```suggestion
    <div class="icons">
      <a routerLink=""><mat-icon class="check-icon">check</mat-icon></a>
      <a routerLink=""><mat-icon class="edit-icon">edit</mat-icon></a>
      <button type="button" (click)="deleteOffreDeStage()"><mat-icon class="close-icon">close</mat-icon></button>
    </div>
```

Reply:
Allo, je vais ajouter le bouton et les fonctionnalités dans la PSA 20 Offre de stage détaillé / Fonctionnalités